### DISCUSSION TITLE
blog/2023/09/09/setup-sdl-with-cmake-and-cpm/

### DISCUSSION THREAD

**Discussion by giscus** [InfiniBrains/Awesome-GameDev-Resources#32] [General]
# blog/2023/09/09/setup-sdl-with-cmake-and-cpm/

Awesome GameDev Resources for learning how to make games

https://courses.tolstenko.net/blog/2023/09/09/setup-sdl-with-cmake-and-cpm/

**Comment by jnetu** [comment]
thanks for the guide. I will start learning c++ game development to make 2D projects and small 3D projects(idk) using SDL

**Reply by tolstenko** [reply]
If you need help, just shoot it
